Terasology/Apiculture
# Add bees

Add placeholder items for bees, and ensure major attributes (sex, genetics) are present and testable.

To do:
- [x] Add bee items
- [x] Add mating
- [x] Add mutations

Testing (Out of date):
- Spawn prefabs "bee_a" and "bee_b"
- Set mate target by holding one of the bees and using "beeMateTarget"
    - Ensure non-bees/empty slots cannot be targeted
- Mate bees by holding the other bee and using "beeMate"
    - Ensure non-bees/empty slots cannot be mated
    - Ensure this command fails gracefully or succeeds in unusual conditions (e.g. target in chest)
- Show genetics of mated bee using "beeDumpGenes"
- Create children using "beeBirth"
     - Ensure count parameter functions as expected
- Examine genetics of children. The active genes should all be from the mate target, and the inactive ones from the bee consumed in mating
- Form a second generation, ideally using one prefab bee and one from the first generation
- Examine genetics of second generation. Results should be consistent with either the active or inactive genes all coming from the prefab bee based on how the commands were used, and a random selection of genes resulting in the other set of genes
